packages/schema/src/config/build.ts

```diff
@@ -38,6 +39,20 @@ export default defineUntypedSchema({
     },
   },
 
+  /**
+   * Whether or not to suppress non-essential logs when building Nuxt.
+   *
+   * Defaults to true when running in CI or when a TTY is not available.
+   *
+   * @type {boolean}
+   * @default {false}
+   */
+  quiet: {
```

Comment:
We could generalize this option to something like `logLevel` (defaulting to quite for CI or when CLI specifies)

Reply:
agreed 👍

Reply:
Yes! We can define levels using an internal enum

Reply:
We talked about it with @danielroe and we came up with an idea to only work with quiet. If the user wants to have different type of logs, he can still use it through vite/webpack configuration in nuxt.config.ts.

What do you think Pooya?

Reply:
(This conversation was early on.) I think supporting multiple log levels is good, with escape hatch of direct vite/webpack configuration.